lib/Raven/Client.php

```diff
@@ -610,8 +610,10 @@ public function captureException($exception, $data = null, $logger = null, $vars
     {
         $has_chained_exceptions = PHP_VERSION_ID >= 50300;
 
-        if (in_array(get_class($exception), $this->exclude)) {
-            return null;
+        foreach ($this->exclude as $exclude) {
+            if (is_a($exception, $exclude)) {
```

Comment:
Why not use `instanceof`?

Reply:
Because `instanceof` works with the literal class name, not a string.

i.e. `$a instanceof Exception` works but `$a instance of 'Exception'` doesnt

Reply:
ha, I'm wrong, 

`$a instanceof 'Exception'` doesnt work work,
but `$c = 'Exception'; $a instanceof $c` does... php...